packages/services/api/src/modules/schema/providers/schema-manager.ts

```diff
@@ -233,7 +233,9 @@ export class SchemaManager {
   @atomic(stringifySelector)
   async getMaybeSchemasOfVersion(schemaVersion: SchemaVersion) {
     this.logger.debug('Fetching schemas (schemaVersionId=%s)', schemaVersion.id);
-    return this.storage.getSchemasOfVersion({ versionId: schemaVersion.id });
+    // !note: The decorators are doing something to prevent passing in an additional boolean for "includeMetadata"...
+    //  so for now always include metadata
+    return this.storage.getSchemasOfVersion({ versionId: schemaVersion.id, includeMetadata: true });
```

Comment:
I'm not sure why the metadata was excluded initially from some calls. I assume the push was for fetching metadata from the CDN and therefore it may have been removed from the schema? Any additional context anyone has on this would help.